packages/simcore-sdk/src/simcore_sdk/node_ports/_item.py

```diff
@@ -1,25 +1,29 @@
 import logging
 import shutil
 from pathlib import Path
-from typing import Dict, Union
+from typing import Callable, Dict, Optional, Union
 
 from yarl import URL
 
-from . import config, data_items_utils, exceptions, filemanager
+from ..node_ports_common import config, data_items_utils, exceptions, filemanager
 from ._data_item import DataItem, DataItemValue
 from ._schema_item import SchemaItem
 
 log = logging.getLogger(__name__)
 
 
-ItemConcreteValue = Union[int, float, bool, str, Path]
-NodeLink = Dict[str,str]
-StoreLink = Dict[str,str]
-DownloadLink = Dict[str,str]
+ItemConcreteValue = Union[int, float, bool, str, Path, Dict[str, str]]
+NodeLink = Dict[str, str]

```

Comment:
Would it be worth to alias the type `Dict[str, str]`?

Reply:
Ah sorry now see this line was not changed ...

Reply:
yes you are right. this is a pretty old piece of code. the node ports package v1 I do not want to change too much.